.github/workflows/ci.yml

```diff
@@ -1,8 +1,15 @@
 ---
 name: "CI"
 on:  # yamllint disable
-  - "push"
-  - "pull_request"
+  push:
+  pull_request:
+    paths-ignore:
+      - "nautobot/docs/**"
+      - "examples/**"
+      - "**.md"
+      - "development/**"
+      - ".github/**"
+      - ".devcontainer/**"
```

Comment:
I would think a change to the `.github/workflows/ci.yml` should *absolutely* trigger CI so that we can verify that it works?